sematic/ui/packages/common/src/pages/RunDetails/PipelineSection.tsx

```diff
@@ -0,0 +1,64 @@
+import styled from '@emotion/styled';
+import Typograph from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import MoreVertButton from 'src/component/MoreVertButton';
+
+import theme from 'src/theme/new';
+
+const HeadLine = styled.h2`
+    margin: 0;
+`;
+
+const TopSection = styled.section`
+    height: 100px;
+    position: relative;
+`;
+
+const BottomSection = styled.section`
+    height: 100px;
+    border-bottom: 1px solid ${theme.palette.p3border.main};
+    display: flex;
+    margin: 0 -25px;
+    & .MuiBox-root {
+        width: 100px;
+        height: 100%;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+    }
+`;
+
+const StyledVertButton = styled(MoreVertButton)`
+    position: absolute;
+    right: 0;
+    top: 12px;
+    transform: translate(50%,0);
+`;
+
+const PipelineSection = () => {
+    return <>
+        <TopSection>
+            <HeadLine>PIPELINE</HeadLine>
```

Comment:
Can we use a CSS style to upcase instead of manually? It will make it easier to modify in the future.